packages/components/date-picker/__tests__/date-picker.test.tsx

```diff
@@ -458,4 +458,122 @@ describe("DatePicker", () => {
       expect(getTextValue(combobox)).toBe("2/4/2019"); // uncontrolled
     });
   });
+
+  describe("Month and Year Picker", () => {
+    const onHeaderExpandedChangeSpy = jest.fn();
+
+    afterEach(() => {
+      onHeaderExpandedChangeSpy.mockClear();
+    });
+
+    it("should show the month and year picker (uncontrolled)", async () => {
```

Comment:
`async` seems unnecessary here. (also for other tests under this describe group)